### PR TITLE
Feature: add ShowInAdmin trait to fields api

### PR DESCRIPTION
**Pull request by jonwaldstein**
## Description

This adds a new `ShowInAdmin` trait to the fields api.

```
$field->shouldShowInAdmin() // false;
$field->showInAdmin(true);
$field->shouldShowInAdmin() // true;
```

## Affects

Existing fields

## Visuals

N/A

## Testing Instructions

The only way to test would be programmatically using the fields api.

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

**Comment by JasonTheAdams** [comment]
@jonwaldstein Why not just put the trait on the `Field` base class? I would think even a hidden field can have the option (and would be false by default).

**Comment by JasonTheAdams** [comment]
Hahahah! Nailed it. 😆

**Comment by jonwaldstein** [comment]
@JasonTheAdams great timing, what about `StoreAsMeta` also?  That's being implemented per-field right now along with `ShowInReceipt`

**Comment by JasonTheAdams** [comment]
Yup, I think both of these can live right on the `Field` class. I can't think of a scenario where they wouldn't at least be sensible in every field type. What makes a field a field is that it has a value.

File: src/Framework/FieldsAPI/Textarea.php [hunk 1]
```diff
@@ -13,8 +13,8 @@ class Textarea extends Field
     use Concerns\HasHelpText;
     use Concerns\HasLabel;
     use Concerns\HasPlaceholder;
-    use Concerns\ShowInReceipt;
-    use Concerns\StoreAsMeta;
+
+
```
Comment (JasonTheAdams) [inline src/Framework/FieldsAPI/Textarea.php:17]: Some formatting fun here

**Review by JasonTheAdams — APPROVED** [review]
Love how much this cleans up. 👍

**Pull request merged**